components/MembershipDriveHeader/MembershipDriveHeader.module.css

```diff
@@ -0,0 +1,134 @@
+.header_container {
+  display: flex;
+  flex-direction: row;
+  height: 100vh;
+}
+
+.title {
+  padding: 8vw 8vw;
+}
+
+.header_title {
+  font-family: var(--title-font);
+  font-weight: 400;
+  font-size: 6vw;
+  letter-spacing: 4px;
+}
+
+.header_button {
+  font-family: var(--main-font);
+  background-color: var(--primary-4);
+  border: none;
+  border-radius: 4px;
+  color: #ffffff;
+  padding: 1vw 2.5vw;
+  margin-top: 3vw;
+}
+
+.header_button:hover {
+  background-color: #ffffff;
+  color: var(--primary-4);
+  transition: all 0.5s ease;
+}
+
+.dotted_line {
+  width: 45vw;
+  position: absolute;
+  height: 1px !important;
+  background-image: linear-gradient(
+      90deg,
+      transparent,
+      transparent 50%,
+      var(--primary-1) 50%,
+      var(--primary-1) 100%
+    ),
+    linear-gradient(to right, rgba(255, 215, 0, 1), rgba(255, 215, 0, 0));
+  background-size: 20px 3px, 100% 3px;
+  border: none;
+  opacity: 1 !important;
+}
+
+.dotted_line_vertical {
+  width: 45vh;
+  position: absolute;
+  height: 1px !important;
+  background-image: linear-gradient(
+      90deg,
+      transparent,
+      transparent 50%,
+      var(--primary-1) 50%,
+      var(--primary-1) 100%
+    ),
+    linear-gradient(to bottom, rgba(255, 215, 0, 1), rgba(255, 215, 0, 0));
+  background-size: 20px 3px, 100% 3px;
+  border: none;
+  opacity: 1 !important;
+  transform: rotate(270deg);
+} 
+
+.image_container {
+  margin: 6vw 2.5vw;
+  height: 45vh;
+  width: 40vw;
+  /* position: relative; */
+}
+
+.code_container {
+  display: inline;
+  position: relative;
+  height: 60vh;
+  float: right;
+}
+
+.sidebar {
+  transform: rotate(-7deg);
+  width: 3.5vw;
+  height: 25vw;
+  margin: 0 2.2vw;
+  /* padding: 3vw; */
+}
+
+/* .code_container{
+    display: inline;
+    position: relative;
+} */
+
+.code {
+  transform: rotate(-7deg);
+  vertical-align: bottom;
+  width: 30vw;
+  height: 25vw;
+  /* padding: 3vw; */
+  /* position: relative; */
+}
+
+.index_tab {
+  position: absolute;
+  transform: rotate(-7deg);
+  border-radius: 2px;
+  box-shadow: 0px 12px 24px rgba(0, 0, 0, 0.34);
+  height: 2.5vw;
+  width: 8vw;
+  top: -8px;
+  left: 0;
+}
+
+/* .index_tab{
+    position: absolute;
+    transform: rotate(-7deg);
+    border-radius: 4px;
+    height: 20px;
+    width: 100px;
+    background-color: #202020;
+    color: #FFFFFF;
+    font-weight: 100;
+    font-size: 14px;
+    padding: 0px 9px;
+    top: -11vw;
+    left: 0;
+    cursor: default;
+}
+
+.tab_cross{
+    display: flex;
+} */
```

Comment:
Move the css file to styles folder and not in components
Also remove all unwanted styles or comments